xtest/runner.py

```diff
@@ -133,14 +170,14 @@ def gen_files(serial):
     return ct, rt, mf
 
 
-def encrypt(sdk, pt_file, ct_file, mime_type="application/octet-stream"):
-    c = [sdk, "encrypt", pt_file, ct_file, "--mimeType", mime_type]
+def encrypt(sdk, pt_file, ct_file, mime_type="application/octet-stream", nano=False):
+    c = [sdk, "encrypt", pt_file, ct_file, str(nano), "--mimeType", mime_type]
```

Comment:
Instead of true/false, maybe 
```suggestion
    c = [sdk, "encrypt", pt_file, ct_file, "nano" if nano else "ztdf", "--mimeType", mime_type]
```
and make the 5th parameter 'container format'. this would be easier to read